cabal.project

```diff
@@ -28,20 +28,129 @@ package *
   ghc-options: -haddock
   test-show-details: direct
 
+source-repository-package
+  type: git
+  location: https://github.com/jwaldmann/blaze-textual.git
+  tag: d8ee6cf80e27f9619d621c936bb4bda4b99a183f
+  -- https://github.com/jwaldmann/blaze-textual/commit/d8ee6cf80e27f9619d621c936bb4bda4b99a183f
+  -- https://github.com/bos/blaze-textual/issues/13
+
+source-repository-package
+  type: git
+  location: https://github.com/mithrandi/czipwith.git
+  tag: b6245884ae83e00dd2b5261762549b37390179f8
+  -- https://github.com/lspitzner/czipwith/pull/2
+
+
+source-repository-package
+  type: git
+  location: https://github.com/jneira/hie-bios/
+  tag: 9b1445ab5efcabfad54043fc9b8e50e9d8c5bbf3
+  -- https://github.com/mpickering/hie-bios/pull/285
+
+source-repository-package
+  type: git
+  location: https://github.com/hsyl20/ghc-api-compat
+  tag: 6178d75772c7d923918dfffa0b1f503dfb36d0a6
+
+source-repository-package
+  type: git
+  location: https://github.com/anka-213/th-extras
```

Comment:
are all these `source-repository-package` entries needed here, or can they be moved to the `cabal-ghc901.project` file?

Reply:
Most or all of them can probably be removed. They are a remnant from before I separated the `cabal.project` file for ghc9.

Reply:
Please remove before merging

Reply:
About these package overrides: does the 8.10 build of ghcide work without them? 

If the answer to 1 is Yes, then I think we should just remove them. 
If not, then I think we need to look into adding more `MIN_PACKAGE_VERSION` pragmas until the answer is Yes. I'm happy to help with that if you give me access to your fork.

Reply:
I think so. I'll try removing them and check that if still builds.

Reply:
In fact, that's already the case for the stack builds, so removing them should be perfectly safe. I just didn't know how to make a separate `cabal.project` file for ghc9. (We'll probably want to merge them together again as soon as all the patches have been merged upstream, so the overrides are no longer needed)